packages/elements/src/containers/API.tsx

```diff
@@ -73,14 +73,30 @@ export interface CommonAPIProps extends RoutingProps {
    * @default false
    */
   hideExport?: boolean;
+
+  /**
+   * Url of a CORS proxy that will be used to send requests in TryIt.
+   * Provided url will be prepended to an URL of an actual request.
+   * @default false
+   */
+  tryItCorsProxy?: string;
 }
 
 const propsAreWithDocument = (props: APIProps): props is APIPropsWithDocument => {
   return props.hasOwnProperty('apiDescriptionDocument');
 };
 
 export const APIImpl: React.FC<APIProps> = props => {
-  const { layout, apiDescriptionUrl = '', logo, hideTryIt, hideSchemas, hideInternal, hideExport } = props;
+  const {
+    layout,
+    apiDescriptionUrl = '',
+    logo,
+    hideTryIt,
+    hideSchemas,
+    hideInternal,
+    hideExport,
+    tryItCorsProxy,
```

Comment:
@pamgoodrich can we document this in the elements config here please: https://meta.stoplight.io/docs/elements/ZG9jOjMyNjU4OTY1-getting-started-with-elements-in-react#configuration